addons/calendar/static/tests/calendar_notification.test.js

```diff
@@ -0,0 +1,94 @@
+import { test } from "@odoo/hoot";
+import { defineCalendarModels } from "@calendar/../tests/calendar_test_helpers";
+import {
+    assertSteps,
+    click,
+    contains,
+    defineMailModels,
+    start,
+    startServer,
+    step,
+} from "@mail/../tests/mail_test_helpers";
+import { onRpc, serverState } from "@web/../tests/web_test_helpers";
+import { registry } from "@web/core/registry";
+
+defineMailModels();
+defineCalendarModels();
```

Comment:
You can call `defineMailModels` inside `defineCalendarModels`

Reply:
Waitinf for constant in mail/